bedrock/firefox/redirects.py

```diff
@@ -471,6 +471,7 @@ def firefox_channel(*args, **kwargs):
     redirect(r"^firefox/android/?$", "firefox.browsers.mobile.android"),
     redirect(r"^firefox/focus/?$", "firefox.browsers.mobile.focus"),
     redirect(r"^firefox/ios/?$", "firefox.browsers.mobile.ios"),
+    redirect(r"^firefox/get-ios/?$", "firefox.browsers.mobile.get-ios"),
```

Comment:
I don't think this is necessary because the link has to be specific for the universal link thing to work, and these others were old URLs that we moved, but I guess it also can't hurt.

Reply:
But if we do leave the redirect we'll also want to add a redirect test.

Reply:
noted! i can just leave it out since it's a specific page